packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "salesforcedx-vscode-lwc",
-  "displayName": "Lightning Code Editor for Visual Studio Code",
-  "description": "Provides code-editing features for Lightning components",
+  "displayName": "LWC Code Editor for Visual Studio Code",
```

Comment:
@esalman-sfdc - Just FYI. To be more consistent with the other naming, I'm just going to use LWC throughout.